gensrc/thrift/AgentService.thrift

```diff
@@ -210,6 +210,8 @@ struct TCloneReq {
     9: optional i64 dest_path_hash;
     10: optional i32 timeout_s;
     11: optional Types.TReplicaId replica_id = 0
+    12: optional i64 table_id
```

Comment:
12 for partition_id, 13 for table_id, keep same order as master and 2.0

Reply:
done